docs/conceptual/cryptography/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Cryptography
+---
+
+This is a high-level overview of the cryptography used in Polkadot.
```

Comment:
Why are we describing specifics of Polkadot in devhub?

Reply:
Some of the text for this PR was taken from the Polkadot wiki. It could be better to generalize it.

Reply:
Yes, right now it is just a copy paste from the polkadot page. I need to redo the whole page.

I might save it for another PR.